server/notebooks/views.py

```diff
@@ -88,6 +89,7 @@ def notebook_revisions(request, pk):
         'date': revision.created.isoformat()}
         for revision in NotebookRevision.objects.filter(notebook_id=pk)])
     return render(request, '../templates/index.html', {
+            'title': f'Revisions - {nb.title}',
```

Comment:
I didn't see a matching test for this change?

Reply:
No, not yet... we actually don't have a test for this view yet. I should probably do the hard thing and add one.

Reply:
Done now